0.wave1_data/1.cellprofiler_ic_processing/README.md

```diff
@@ -0,0 +1,46 @@
+# Create initial LoadData csv to calculate illumination correction function
+
+In this module, we present the process for creation of a LoadData csv and CellProfiler pipeline for calculating an illumination correction function for each channel.
+
+## pe2loaddata
+
+To create the LoadData csv file for CellProfiler, we use a software called [pe2loaddata](https://github.com/broadinstitute/pe2loaddata/tree/220ac512bfc0c2e582d379b19411c1585272aee3). 
+This will create a LoadData csv from the Phenix metadata XML file generated from data acquisition. 
+
+## Step 1: Create CellProfiler environment
+
+If you do not already have CellProfiler installed in a conda environment, you can run the code below to create one.
+
+```sh
+conda env create -f cellprofiler_environment.yml
+```
+
+We are using CellProfiler v4.2.4 for all pipelines.
+
+## Step 2: Install pe2loaddata
+
+Using the instructions from their Github, you can install pe2loaddata by running the code block below in the `0.wave1_data/1.cellprofiler_ic_processing/` directory.
+
+```sh
+git clone https://github.com/broadinstitute/pe2loaddata.git
```

Comment:
We should probably fix the version we're installing. Is there a way to install a specific github hash?

Reply:
@gwaybio 

We might be able to install using this I found from here (https://stackoverflow.com/questions/13685920/install-specific-git-commit-with-pip).

```sh
pip install git+https://github.com/broadinstitute/pe2loaddata.git@0426dd6b9d8b3242294a6fbdef7c4e1ee762a4cc
```